plenum/server/replica.py

```diff
@@ -745,7 +745,7 @@ def create3PCBatch(self, ledger_id):
             rv = self.execute_hook(ReplicaHooks.CREATE_PPR, pre_prepare)
             pre_prepare = rv if rv is not None else pre_prepare
 
-        self.logger.debug('{} created a PRE-PREPARE with {} requests for ledger {}'.format(
+        self.logger.trace('{} created a PRE-PREPARE with {} requests for ledger {}'.format(
```

Comment:
Do we have other DEBUG level entries where we log creation of a PRE-PREPARE by a Primary?

Reply:
We have at least sends for any message logged, is it enough?

Reply:
Yes, absolutely